packages/design-system/src/components/OcResource/OcResource.vue

```diff
@@ -165,9 +165,14 @@ export default {
       }
     },
 
+    isTxtFileAlmostEmpty() {
```

Comment:
IMO this logic doesn't belong into the `OcResource` component - it already has a prop whether or not the thumbnail should be displayed. We should make use of the prop from wherever we embed the `OcResource` component (currently in `ResourceTable.vue` and search `Preview.vue`). Ping me if you want to work on this together.

Reply:
alright